src/nacl/bindings/crypto_kx.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2018 Donald Stufft and individual contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
+
+from nacl import exceptions as exc
+from nacl._sodium import ffi, lib
+from nacl.exceptions import ensure
+
+__all__ = ["crypto_kx_keypair",
+           "crypto_kx_client_session_keys",
+           "crypto_kx_server_session_keys",
+           "crypto_kx_PUBLIC_KEY_BYTES",
+           "crypto_kx_SECRET_KEY_BYTES",
+           "crypto_kx_SEED_BYTES",
+           "crypto_kx_SESSION_KEY_BYTES"]
+
+"""
+Implementations of client, server key exchange
+"""
+crypto_kx_PUBLIC_KEY_BYTES = lib.crypto_kx_publickeybytes()
+crypto_kx_SECRET_KEY_BYTES = lib.crypto_kx_secretkeybytes()
+crypto_kx_SEED_BYTES = lib.crypto_kx_seedbytes()
+crypto_kx_SESSION_KEY_BYTES = lib.crypto_kx_sessionkeybytes()
+
+
+def crypto_kx_keypair(seed=None):
+    """
+    Generate a keypair.
+    :param seed: random seed
+    :type seed: bytes
+    :return: (public_key, secret_key)
+    :rtype: (bytes, bytes)
+    """
```

Comment:
I think a comment about the keypair really being a x25519 keypair should be added, either inside the docstring or before the implementation.